src/main/java/com/dumbdogdiner/stickyapi/common/util/textures/InvalidTextureException.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util.textures;
+
+import java.text.MessageFormat;
+
+/*
+ * Exception
```

Comment:
A more explicit description of what this error is for would be lovely - I know it seems obvious but I'm a dummy fox and will get this wrong.